[4.3.2] - Cant open Player Charakter Sheets

Hello,
After upgrading to the latest version, I can no longer open all character sheets. I can open some of them, but I haven't found a connection yet.
- All modules are deactivated

I get the following error message (FireFox):
`Error: An error occurred while rendering ActorSheet5eCharacter2 26. item.labels is undefined
Caused by: TypeError: item.labels is undefined
    _prepareItem sheet-v2-mixin.mjs:481
    <anonymous> character-sheet.mjs:105
    _prepareItems character-sheet.mjs:66
    _prepareItems sheet-v2-mixin.mjs:344
    _prepareItems character-sheet-2.mjs:264
    getData base-sheet.mjs:195
    getData character-sheet.mjs:24
    getData sheet-v2-mixin.mjs:159
    getData sheet-v2-mixin.mjs:99
    getData character-sheet-2.mjs:108
    _render foundry.js:5838
    _render foundry.js:6572
    _render foundry.js:7158
    _render sheet-v2-mixin.mjs:67
    _render sheet-v2-mixin.mjs:79
    _render character-sheet-2.mjs:80
    render foundry.js:5793
    _onClickEntryName foundry.js:73976
    jQuery 8
    activateListeners foundry.js:73165
    activateListeners foundry.js:73964
    _render foundry.js:5867
    _render foundry.js:72780
    _render foundry.js:73125
    _render foundry.js:73922
    _render foundry.js:74626
    render foundry.js:5793
    initializeUI foundry.js:9245
    setupGame foundry.js:9083
    _initializeGameView foundry.js:10432
    _initializeView foundry.js:10408
    initialize foundry.js:8995
    async* foundry.js:95974
    async* foundry.js:95954
foundry.js:655:30
`

Apparently the error is during data preparation :
`Error: Failed data preparation for Actor.pbsoIarnoAlbrek0. Invalid number of embedded UUID parts
Caused by: Error: Invalid number of embedded UUID parts
    parseUuid foundry-esm.js:3571
    parseUuid parse-uuid.mjs:10
    get actor.mjs:3562
    prepareFinalData base-activity.mjs:588
    prepareFinalActivityData activities.mjs:300
    prepareFinalData feat.mjs:121
    prepareFinalAttributes item.mjs:673
    prepareData actor.mjs:204
    forEach foundry-esm.js:5031
    prepareData actor.mjs:204
    _safePrepareData foundry.js:10799
    initializeDocuments foundry.js:9168
    setupGame foundry.js:9054
    _initializeGameView foundry.js:10432
    _initializeView foundry.js:10408
    initialize foundry.js:8995
    async* foundry.js:95974
    async* foundry.js:95954
foundry.js:655:30
`
`Error: Failed data preparation for Actor.M2zkDxPCI79B5UT4.Item.fAJvwGRvMAAb1hKM. parsed.collection is undefined
    onError foundry.js:654
    _safePrepareData foundry.js:10801
    prepareEmbeddedDocuments foundry.js:10845
    prepareEmbeddedDocuments foundry.js:15879
    prepareEmbeddedDocuments actor.mjs:225
    prepareData foundry.js:10822
    prepareData foundry.js:15863
    prepareData flags.mjs:23
    prepareData actor.mjs:203
    _safePrepareData foundry.js:10799
    initializeDocuments foundry.js:9168
    setupGame foundry.js:9054
    _initializeGameView foundry.js:10432
    _initializeView foundry.js:10408
    initialize foundry.js:8995
    async* foundry.js:95974
    async* foundry.js:95954
Caused by: TypeError: parsed.collection is undefined
    parseUuid parse-uuid.mjs:13
    set actor.mjs:3577
    prepareBaseData abstract.mjs:509
    prepareData foundry.js:10820
    prepareData flags.mjs:23
    _safePrepareData foundry.js:10799
    prepareEmbeddedDocuments foundry.js:10845
    prepareEmbeddedDocuments foundry.js:15879
    prepareEmbeddedDocuments actor.mjs:225
    prepareData foundry.js:10822
    prepareData foundry.js:15863
    prepareData flags.mjs:23
    prepareData actor.mjs:203
    _safePrepareData foundry.js:10799
    initializeDocuments foundry.js:9168
    setupGame foundry.js:9054
    _initializeGameView foundry.js:10432
    _initializeView foundry.js:10408
    initialize foundry.js:8995
    async* foundry.js:95974
    async* foundry.js:95954
foundry.js:655:30
`

`Error: Failed data preparation for Actor.M2zkDxPCI79B5UT4. this.labels is undefined
 
Caused by: TypeError: this.labels is undefined
    _prepareLabels item.mjs:684
    prepareFinalAttributes item.mjs:674
    prepareData actor.mjs:204
    forEach foundry-esm.js:5031
    prepareData actor.mjs:204
    _safePrepareData foundry.js:10799
    initializeDocuments foundry.js:9168
    setupGame foundry.js:9054
    _initializeGameView foundry.js:10432
    _initializeView foundry.js:10408
    initialize foundry.js:8995
    async* foundry.js:95974
    async* foundry.js:95954
foundry.js:655:30
`

--- a/module/parse-uuid.mjs
+++ b/module/parse-uuid.mjs
@@ -8,6 +8,7 @@ export default function parseUuid(uuid, {relative}={}) {
   if ( !uuid ) throw new Error("A UUID string is required.");
   if ( uuid.startsWith(".") && relative ) return _resolveRelativeUuid(uuid, relative);
   const parsed = foundry.utils.parseUuid(uuid, {relative});
+  if ( !parsed?.collection ) return parsed;
   const remappedUuid = uuid.startsWith("Compendium") ? [
     "Compendium",
     parsed.collection.metadata.id,

